beta/src/pages/learn/state-a-components-memory.md

```diff
@@ -151,54 +151,54 @@ button {
 
 </Sandpack>
 
-The `handleClick()` event handler is updating a local variable, `index`. But two things prevent that change from being visible:
+`handleClick()` event handler एक लोकल वैरिएबल, `इंडेक्स` को अपडेट कर रहा है। लेकिन दो चीजें उस बदलाव को दिखने से रोकती हैं
 
-1. **Local variables don't persist between renders.** When React renders this component a second time, it renders it from scratch—it doesn't consider any changes to the local variables.
-2. **Changes to local variables won't trigger renders.** React doesn't realize it needs to render the component again with the new data.
+1. **लोकल वैरिएबल रेंडरर्स के बीच बने नहीं रहते हैं** जब React इस कौम्पोनॅन्ट को दूसरी बार रेंडर करता है, यह इसे नए सिरे से रेंडर करता है—यह लोकल वैरिएबल में किसी भी परिवर्तन पर विचार नहीं करता है।
+2. **लोकल वैरिएबल में बदलाव करने से रेंडर ट्रिगर नहीं होंगे.** रिएक्ट को एहसास नहीं होता है कि उसे नए डेटा के साथ कौम्पोनॅन्ट को फिर से रेंडर करने की आवश्यकता है।
 
-To update a component with new data, two things need to happen:
+किसी कौम्पोनॅन्ट को नए डेटा से अपडेट करने के लिए, दो चीज़ें होनी चाहिए:
 
-1. **Retain** the data between renders.
-2. **Trigger** React to render the component with new data (re-rendering).
+1. रेंडरर्स के बीच डेटा को **बनाए रखें**।
+2. कौम्पोनॅन्ट को नए डेटा (पुनः प्रतिपादन) के साथ रेंडर करने के लिए React को **प्रेरित करे**।
 
-The [`useState`](/apis/usestate) Hook provides those two things:
+यह दोनों चीज़े हुक [`useState`](/apis/usestate) की सहायता से की जा सकती है::
 
-1. A **state variable** to retain the data between renders.
-2. A **state setter function** to update the variable and trigger React to render the component again.
+1. एक **state वैरिएबल** रेंडरर्स के बीच डेटा को बनाए रखे।
+2. एक **state setter फंक्शन (सेटर-कार्य)** वैरिएबल अद्यतन करने के लिए और कौम्पोनॅन्ट को फिर से प्रस्तुत करने के लिए react को प्रेरित करें।
 
-## Adding a state variable {/*adding-a-state-variable*/}
+## एक state वैरिएबल जोड़ना {/*adding-a-state-variable*/}
 
-To add a state variable, import `useState` from React at the top of the file:
+एक state वैरिएबल जोड़ने के लिए, आयात करे `useState` react मॉड्यूल से, फ़ाइल के शीर्ष पर:
```

Comment:
```
आयात करे `useState` react मॉड्यूल से, फ़ाइल के शीर्ष पर:
```
=>
```
फाइल से सबसे ऊपर `useState` को React से इम्पोर्ट करें:
```

Reply:
I think फाइल से सबसे ऊपर `useState` को React से इम्पोर्ट करें: is much better than फाइल **में** सबसे ऊपर `useState` को React से इम्पोर्ट करें:. Let me know

Reply:
Yes. You are right. 

```
 फाइल से सबसे ऊपर useState को React से इम्पोर्ट करें:
```

sounds better. Can you update this.